zanata-war/src/main/java/org/zanata/adapter/OkapiFilterAdapter.java

```diff
@@ -172,6 +168,9 @@ public Resource parseDocumentFile(URI documentContent,
         RawDocument rawDoc =
                 new RawDocument(documentContent, "UTF-8",
                         net.sf.okapi.common.LocaleId.fromString("en"));
+        if (rawDoc.getTargetLocale() == null) {
+            rawDoc.setTargetLocale(net.sf.okapi.common.LocaleId.ENGLISH);
```

Comment:
Source defaulting to English: yes, to some extent (but we should try to put that default in exactly one place, instead of all over the codebase).

But this seems to be target locale.